server/test/repository/StoredFileRepositoryTest.java

```diff
@@ -64,4 +67,23 @@ public void lookupFile() {
 
     assertThat(result).isEqualTo(file);
   }
+
+  /**
+   * This is a regression test to ensure that files created before the migration to stored file ACLs
```

Comment:
Related, if this is going to stick around after the migration then it should be commented to make sense when someone reads it in 6 months.